gitless/cli/gl_fetch.py

```diff
@@ -6,6 +6,7 @@
 
 
 from ..import core
+from . import helpers, pprint
```

Comment:
The `helpers` import is not used here. Could you remove please?

Same applies to:
- [ ] gl_pull.py
- [ ] [gl_patch.py](https://github.com/goldstar611/gitless/pull/33/files#diff-e7155535a5cadd32eeb2bb38c5fcc31228792b6005b7b6c64fef2f2f29efd65d)